www/class/centreonService.class.php

```diff
@@ -78,6 +78,29 @@ public function __construct($db, $dbMon = null)
         $this->instanceObj = new CentreonInstance($db);
     }
 
+    /**
+     * filteredArrayId
+     *
+     * @param  int[] $ids
+     * @return int[] filtered
+     */
+    function filteredArrayId(array $ids): array
+    {
+        /* Slight difference here. Array parameter is made
+         * of combined ids HOSTID_SERVICEID
+         */
+        $combinedIds = [];
+        return array_filter($ids, function ($combinedId) {
+            $combinedIds = explode("_", $combinedId);
```

Comment:
you may need to check that $combinedId has a "_" using strpos before the explode or use regexp instead of explode
(to avoid warning or notice from the explode method)

Reply:
I've choosen the `preg_match` method :smiley: